### DISCUSSION TITLE
Akka cluster libraries

### DISCUSSION THREAD

**Discussion by taher-cldcvr** [apache/pekko#503] [Q&A]
Hi 

Where can i find 

akka-management-cluster-bootstrap
akka-discovery-kubernetes-api
akka-management-cluster-http
akka-cluster-metrics
akka-cluster-sharding

All of these libraries I cannot find. Please help

**Comment by pjfanning** [comment]
Apache Pekko is separate from Akka. If you want to find about Akka, you should read the Akka docs.

See https://doc.akka.io/docs/akka-management/current/

**Reply by taher-cldcvr** [reply]
I know that Pekko is a fork of Akka because Akka changed its license. So when we develop something on Pekko that Akka did, we used these libs. I want to know what is the equivalent of these libs in Pekko

**Reply by pjfanning** [reply]
see https://pekko.apache.org/docs/pekko-management/current/

Snapshot jars are available and a full pekko-management 1.0.0 release should happen in a few weeks.
